src/tracks/playabletrack/notetrack/ui/NoteTrackView.cpp

```diff
@@ -723,7 +736,20 @@ void NoteTrackView::Draw(
       const auto hasSolo = artist->hasSolo;
       muted = (hasSolo || nt->GetMute()) && !nt->GetSolo();
 #endif
+
+#ifdef EXPERIMENTAL_NOTETRACK_OVERLAY
+      TrackArt::DrawBackgroundWithSelection(context, rect, nt.get(), AColor::labelSelectedBrush, AColor::labelUnselectedBrush);
+      
+      bool selected{ false };
+      if (auto affordance = std::dynamic_pointer_cast<NoteTrackAffordanceControls>(GetAffordanceControls()))
+      {
+         selected = affordance->IsSelected();
+      }
+
+      DrawNoteTrack(context, nt.get(), rect, muted, selected);
+#else
       DrawNoteTrack( context, nt.get(), rect, muted );
```

Comment:
Compilation fails if the EXPERIMENTAL is turned off.  Fix it

Reply:
Thanks! Fixed whole drawing, should be fine now